activities/titilayo_tl.py

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env python3
+
+import os
+
+import grass.script as gs
+
+def run_overlandflow(scanned_elev, env, **kwargs):
+    gs.run_command("v.to.rast", input="streams@PERMANENT", output="rural_streams", use="val", val=1, flags="d")
```

Comment:
```suggestion
    gs.run_command(
        "r.watershed",
        elevation=scanned_elev,
        threshold=5000,
        accumulation="accum",
        drainage="drainage_direction",
        stream="streams",
        flags="a"
    )
    gs.run_command(
        "r.stream.distance",
        stream_rast="streams",
        direction="drainage_direction",
        elevation="elevation",
        method="downstream",
        difference="above_stream"
    )
```

Reply:
This code will create the Height Above Nearest Drainage raster so you can use the streams as a seed. The workflow is from [Assignment 3](https://github.com/ncsu-geoforall-lab/GIS714-assignments/blob/main/Surface_Water_Simulations/WaterSim-FlowAcc-and-HAND.ipynb).